app/api/cruds/meal_crud.py

```diff
@@ -0,0 +1,102 @@
+"""Set up Graphql for Meal Model"""
+import graphene
+from graphene_django.types import DjangoObjectType
+from graphene_django.converter import convert_django_field
+
+from django.db.models import TimeField
+from django.core.exceptions import ValidationError
+
+from app.timetables.models import Meal
+from app.api.cruds.utils import get_errors, get_object, load_object
+
+
+# https://github.com/graphql-python/graphene-django/issues/18
+@convert_django_field.register(TimeField)
+def convert_function(field, registry=None):
```

Comment:
Right now it isn't and I figured that since we don't have any other models with TimeField, it makes sense to keep it here.